src/assets/scss/layout/layout.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../../assets/scss/helpers/variables";
+
+.pf-c-dropdown {
+  &__menu {
+    &-item {
+      color: #000 !important;
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  position: fixed;
+  height: 55px;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  width: 100%;
+
+  .footer-text {
+    color: white;
+    text-align: center;
+  }
+  
+  .footer-link {
+    display: flex;
+    align-items: center;
+    
+  .footer-logo {
+    width: 50px;
+    margin-left: 10px;
+    margin-right: 10px;
+    }
+  }
+}
+
+.footer-styling {
+  margin-bottom: 55px;
+}
```

Comment:
Should we be using `em`s instead of `px` for more responsive sizing?

Reply:
Changed much of the px units to em, but as discussed in call, much of the project is still using px measurements and struggles with responsiveness, so it's best to have proper discussion about how to make COVID-Net more responsive and what units would be best for the project and do a proper overhaul of legacy code